src/layouts/IndexLayout.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import styled from "styled-components";
+import Link from "next/link";
+import TEMP_FEATURED_DATA from "./../../projects.json";
+import ProjectPreview from "./../components/project/ProjectPreview";
+import Button from "../components/general/Button.js";
+
+function renderProjects() {
+  // TEMPORARY DATA
+  return TEMP_FEATURED_DATA.projects.map(project => {
+    return <ProjectPreview data={project} key={project.name}></ProjectPreview>;
+  });
+}
+
+export default function index() {
+  return (
+    <Container>
+      <h1>Share and Collaborate on projects to reach the maximum positive impact on the world</h1>
+
+      <h3>Featured</h3>
```

Comment:
Because there's an `h1` above and `h3`s in `ProjectPreview`, should this be an `h2` instead?

And if the styling on the `h2` is wrong for this spot, we should change the styles, not the tag—it's usually harder to add accessibility later.